develop-seat-adjuster.md

```diff
@@ -138,10 +138,25 @@ The different runtimes are maintained in [a separate Eclipse Velocitas repositor
 we use the [Eclipse Velocitas CLI](https://github.com/eclipse-velocitas/cli):
 
 ```bash
-velocitas exec runtime-kanto up
+velocitas exec runtime-local up
 ```
+The local runtime allows you to debug your python vehicle app code. 
+Thus, in den development phase we opt for this runtime.
+Later, e.g. when you want to run your app in a similar environment as it would run on the target environment (e.g. Eclipse Leda), we can use the Eclipse Kanto runtime here. 
```

Comment:
can you add the hint on how to run the Kanto runtime (Velocitas exec runtime-kanto up)

Reply:
I will merge and add the hint myself.